ios/FluentUI/Navigation/BarButtonItemButton.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class BarButtonItemButton: UIButton {
```

Comment:
The badge still seems to be behind the button label on iPad (iOS 14.5) even though the mask is applied to it:

![badge_behind_issue](https://user-images.githubusercontent.com/68076145/133119521-2013237b-62e3-4c21-b56e-dcc6e4bec892.png)

Reply:
It's been fixed!!